docs/content/contribute/review-pr.md

```diff
@@ -23,6 +23,7 @@ This page provides guidelines for reviewing Magic Modules pull requests
    1. all fields added/updated in the PR appear in at least one test.
       * It is advisable to test updating from a non-zero value to a zero value if feasible.
    1. all mutable fields are tested in at least one update test.
+   1. all resources in the acceptance tests have a `tf-test` or `tf_test` prefix in their primary id field.
```

Comment:
This is automatic for example tests, should we scope it to handwritten ones?

Reply:
this is only automatic for example tests if the name includes a `-` - we should be checking both kinds.